src/assets/wise5/authoringTool/project-authoring-lesson/project-authoring-lesson.component.ts

```diff
@@ -1,18 +1,31 @@
 import { Component, EventEmitter, Input, Output } from '@angular/core';
 import { TeacherDataService } from '../../services/teacherDataService';
+import { TeacherProjectService } from '../../services/teacherProjectService';
+import { SelectNodeEvent } from '../domain/select-node-event';
+import { NodeTypeSelected } from '../domain/node-type-selected';
 
 @Component({
   selector: 'project-authoring-lesson',
   templateUrl: './project-authoring-lesson.component.html',
   styleUrls: ['./project-authoring-lesson.component.scss']
 })
 export class ProjectAuthoringLessonComponent {
-  @Input() disableSelection: boolean = false;
-  @Input() item: any;
-  @Output() selectNodeEvent: EventEmitter<void> = new EventEmitter<void>();
+  checked: boolean = false;
+  idToNode: any = {};
+  @Input() lesson: any;
+  @Input() nodeTypeSelected: NodeTypeSelected;
```

Comment:
ProjectAuthoringLesson and ProjectAuthoringStep shouldn't have to know about NodeTypeSelected- they should only need to know if they need to disable selection?
```suggestion
  @Input() disableSelection: boolean = false;
```